lib/lintPostcssResult.js

```diff
@@ -107,16 +107,27 @@ function lintPostcssResult(stylelintOptions, postcssResult, config) {
 
 		performRules.push(
 			Promise.all(
-				postcssRoots.map((postcssRoot) =>
-					ruleFunction(primaryOption, secondaryOptions, {
-						fix:
-							stylelintOptions.fix &&
-							// Next two conditionals are temporary measures until #2643 is resolved
-							isFileFixCompatible &&
-							!postcssResult.stylelint.disabledRanges[ruleName],
-						newline,
-					})(postcssRoot, postcssResult),
-				),
+				postcssRoots.map(async (postcssRoot) => {
+					try {
+						return await ruleFunction(primaryOption, secondaryOptions, {
+							fix:
+								stylelintOptions.fix &&
+								// Next two conditionals are temporary measures until #2643 is resolved
+								isFileFixCompatible &&
+								!postcssResult.stylelint.disabledRanges[ruleName],
+							newline,
+						})(postcssRoot, postcssResult);
+					} catch (ex) {
+						const fileName =
+							postcssRoot.source && postcssRoot.source.input
+								? postcssRoot.source.input.file
+								: 'unknown file';
+
+						// eslint-disable-next-line no-console
```

Comment:
How do we feel about disabling this rule here?

Reply:
IMO, disabling it here seems no problems. 👍 
The current ESLint configuration may be a bit too strict, so it seems good to me to loosen it via the `allow` option as follows:

```json
{
  "no-console": ["error", { "allow": ["warn", "error"] }]
}
```

https://github.com/stylelint/eslint-config-stylelint/blob/aecb506259e5371e162ee590b67d671d1bc72095/.eslintrc.js#L33

See also <https://eslint.org/docs/rules/no-console#options>

Reply:
About this, I've opened a new issue: https://github.com/stylelint/eslint-config-stylelint/issues/108